docs/source/community_guides.rst

```diff
@@ -5,19 +5,19 @@
 Contributing
 ============
 
-Currently we are not accepting community contributions. Though, we encourage
-opening `git issues`_ for bugs, comments or feature requests.
+We are not currently accepting community contributions. However, we encourage
+you to open `git issues`_ for bugs, comments or feature requests.
 
-Periodically review this content to learn when and how contributions can be
-made in the future.
+Review this content periodically to learn when and how to make contributions in
+the future.
 
 .. _git issues:
    https://github.com/ansible-collections/ibm_zos_core/issues
 
 Helpful Links
 =============
 
-* Getting started `Ansible guide`_.
+* Getting Started `Ansible guide`_.
```

Comment:
Without breaking any link, is it possible to rewrite this link to Ansible Getting Started guide? Getting started ansible guide is a little awkward

Reply:
That is no problem to rename ` Getting Started `Ansible guide`_ .
It is only the anchors and the html page names we want to avoid changing as without notice.

Reply:
agree. let's park this one as well for refactor later